projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension/src/__tests__/deploy-job-component.spec.ts

```diff
@@ -38,19 +38,6 @@ describe('#render()', () => {
   });
 });
 
-describe('#onEnableClick', () => {
```

Comment:
You may want to just add a simple test or extend some assert /verification to ensure that the new default behavior works